addons/l10n_in/models/account_invoice.py

```diff
@@ -310,6 +311,34 @@ def _compute_l10n_in_warning(self):
                         'action_text': action_text,
                     }
 
+            if (
+                move.is_sale_document(include_receipts=False)
+                and company.l10n_in_is_gst_registered
+            ):
+                tags = move.invoice_line_ids.filtered(line_filter_func).tax_tag_ids
+                tax_tag_suffix = [
+                    "base_sgst",
```

Comment:
```suggestion
                    'base_sgst',
```
technical string with `''`?